### PR TITLE
doc: refresh website

### PR DESCRIPTION
**Pull request by sh0rez**
Adds the new Logo to the website and also rebuilds the landing page to
make use of some advanced layouting React gives us.

Still lot to do on the website, but this gets us started

### CODE REVIEW

**Comment by malcolmholmes** [comment]
Can you remind me the preview URL? Thanks!

**Comment by sh0rez** [comment]
> Can you remind me the preview URL? Thanks!

Screen Shot 2020-05-26 at 10 28 30

It's in a CI check ^^

File: docs/docs/introduction.mdx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+## Highlights
+
+<Highlights
+  elems={{
+    Clean:
+      "The Jsonnet language expresses your Kubernetes apps more obviously than YAML ever did",
+    Reusable:
+      "Build application libraries, import them anywhere and even share them on GitHub!",
+    Concise:
+      "Using the Kubernetes library, you will never see boilerplate again!",
```
Comment (trotttrotttrott) [inline docs/docs/introduction.mdx:24]:
```suggestion
      "Using the [Kubernetes library](https://jsonnet-libs.github.io/k8s-alpha/), you will never see boilerplate again!",
```
Not sure if markdown is interpreted here 🤔

File: docs/docs/introduction.mdx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import { Catcher } from "../src/components/home"
+
+<Catcher />
+
+## Highlights
+
+<Highlights
+  elems={{
+    Clean:
+      "The Jsonnet language expresses your Kubernetes apps more obviously than YAML ever did",
```
Comment (trotttrotttrott) [inline docs/docs/introduction.mdx:20]:
```suggestion
      "The Jsonnet language expresses your Kubernetes apps more clearly than YAML ever did",
```

File: docs/docs/introduction.mdx (hunk 1 above)
Comment (sh0rez) [reply docs/docs/introduction.mdx:24]: It's not. Will re-do highlights soon in a way that markdown works again

File: docs/docs/introduction.mdx [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+route: /
+title: Introduction
+
+seoTitle: "Grafana Tanka: Define. Reuse. Override."
+description: >
+  Grafana Tanka is the robust configuration utility for your Kubernetes cluster, powered by the Jsonnet language.
```
[the remark this answers is not in the document]
Comment (sh0rez) [reply docs/docs/introduction.mdx:7]: Not sure about this one. 

I think we want to highlight that Tanka is unique and new, compared to existing solutions like Helm and Kustomize.

Using `a` makes it appear as one of many, while `the` feels slightly aggressive, yet acceptable.

Not a native english speaker though, so I can't finally judge whether this is too offensive for marketing language

[the remark this answers is not in the document]
Comment (RichiH) [reply docs/docs/introduction.mdx:7]: It's bold, but fine.

**Pull request merged**